challenges.js

```diff
@@ -4,81 +4,79 @@ class Challenge {
   }
 
   render() {
-    const challengeCard = document.createElement('div');
+    const challengeCard = document.createElement("div");
```

Comment:
Many of these changes only affect the quote marks. Having such changes in a git diff makes it much more difficult for a reviewer to sort out the actually _meaningful_ changes to the code, increasing the mental load of the reviewer, and hence the risk that they miss something important.

It's a good idea at the beginning of any team project to agree on rules for formatting (e.g. quote marks, indentation etc) and configure editors or other automated software to enforce that formatting. That way, no one will ever commit code that doesn't conform with the rules, and no one will have to change formatting as part of their PRs.